### ISSUE TITLE
pivot option to gate_mindensity and gate_mindensity2 is not honored

### ISSUE THREAD

**Issue by jacobpwagner** [RGLab/openCyto#207]
In constraining the search for the gate boundary returned from `gate_mindensity` or `gate_mindensity2`, the `min`, `max`, and `gate_range` args work as expected. However, the `pivot` option does not. According to the docs:

"pivot | logical value. If TRUE, we choose as the two peaks the largest peak and its neighboring peak. See details."

(details from `mindensity`):
"In the default case, the two peaks of interest are the two largest peaks obtained from the density function. However, if pivot is TRUE, we choose the largest peak and its neighboring peak as the two peaks of interest. In this case, the neighboring peak is the peak immediately to the left of the largest peak if positive is TRUE. Otherwise, the neighboring peak is selected as the peak to the right"

However, this does not seem to work at all and a quick glance at the source suggests that the option is never utilized. Demonstration:
```
library(flowCore)
library(flowWorkspace)
library(openCyto)
library(ggcyto)
gs <- load_gs(system.file("extdata", "gs_bcell_auto", package="flowWorkspaceData"))
fr <- gh_pop_get_data(gs[[1]])
autoplot(fr, getChannelMarker(fr, "CD38")$desc)
```
![image](https://user-images.githubusercontent.com/43548373/74488560-6de4b000-4e77-11ea-9aae-c88bbf410d55.png)

Here, I think it's just finding the global minimum:
```
gate <- gate_mindensity2(fr,
                         channel="CD38",
                         filterID="nonDebris",
                         pivot=TRUE,
                         plot=TRUE
)
gate
```
```
Rectangular gate '' with dimensions:
  CD38: (-335.765135027374,Inf)
```

![image](https://user-images.githubusercontent.com/43548373/74488654-c1ef9480-4e77-11ea-8489-78eaca6c2988.png)

The same call with `pivot=FALSE` results in the exact same result. Of course, using `gate_range` or `min`/`max`, we can get the desired boundary.

```
gate <- gate_mindensity2(fr,
                         channel="CD38",
                         filterID="nonDebris",
                         pivot=FALSE,
                         gate_range = c(800,3000),
                         plot=TRUE
)
gate
```
```
Rectangular gate '' with dimensions:
  CD38: (1497.26841196133,Inf)
```
![image](https://user-images.githubusercontent.com/43548373/74488941-9325ee00-4e78-11ea-8550-1acc93418393.png)

I'm not sure if the option was just never implemented, but we should probably either implement it fully (it would be kind of handy in automated pipelines) or remove it from the args and docs.

**Comment by gfinak** [comment]
@jacobpwagner is this still an open issue?

**Comment by jacobpwagner** [comment]
Yeah, I believe it is still open. It's probably a pretty quick fix, though.  It must have just slipped through the cracks.

**Comment by Pandapip1** [comment]
> Here, I think it's just finding the global minimum:

This is still not fixed. It seems to still be gating around the global min, even on the latest github version:
![image](https://user-images.githubusercontent.com/45835846/126833125-6d04ebf8-6a73-4042-ae81-2743761ce85e.png)

**Comment by jacobpwagner** [comment]
@Pandapip1 , rather than being fully implemented, the option was simply removed in 956808f71b5befa23688552351e2b25b091e7c82, so I think this is expected behavior.
